mantle/cmd/kola/devshell.go

```diff
@@ -299,11 +299,25 @@ func checkWriteState(msg string, c chan<- guestState) {
 }
 
 type systemdEventMessage struct {
-	Unit      string `json:"UNIT"`
-	MessageID string `json:"MESSAGE_ID"`
-	Message   string `json:"MESSAGE"`
-	JobResult string `json:"JOB_RESULT"`
-	JobType   string `json:"JOB_TYPE"`
+	Unit      string      `json:"UNIT"`
+	MessageID string      `json:"MESSAGE_ID"`
+	Message   interface{} `json:"MESSAGE"`
+	JobResult string      `json:"JOB_RESULT"`
+	JobType   string      `json:"JOB_TYPE"`
+}
+
+func (se systemdEventMessage) message() string {
+	if a, ok := se.Message.([]interface{}); ok {
```

Comment:
Shouldn't this be

```suggestion
	if a, ok := se.Message.([]byte); ok {
```

?

Otherwise the byte type assertion we do in the loop will panic.

Reply:
What's emitted into the JSON is an array of decimal numbers.  Go doesn't know it's a byte array, it knows it's a `[]float64`.  Now where I messed up here is casting to `[]interface{}` instead of `[]float64`; fixed that.

And while I'm here I also changed things so we panic if we find an unexpected type; the "return invalid string" thing would be annoying to debug.